gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=22.1.0-RC2-SNAPSHOT
+version=22.10.0-RC3
```

Comment:
s/22.10/22.1

Reply:
Thanks.  I keep doing this for some reason

Reply:
I'm happy otherwise!